lib/jsdom/living/xmlhttprequest.js

```diff
@@ -590,6 +594,30 @@ module.exports = function createXMLHttpRequest(window) {
     DONE: 4
   });
 
+  function parseContentType(contentType) {
```

Comment:
https://www.npmjs.com/package/con-type looks pretty good too; maybe it is better at handling invalid cases.